core/__tests__/models/export.ts

```diff
@@ -714,6 +717,7 @@ describe("models/export", () => {
 
     test("an export needs a valid level", async () => {
       errorExport.errorMessage = "interesting stuff happened!";
+      //@ts-ignore
       errorExport.errorLevel = "other";
       await expect(errorExport.save()).rejects.toThrow(/Validation error/);
     });
```

Comment:
Now that we have strong types, and we test that something is wrong with input, we will need more `@ts-ignores` - like here, "other" isn't a valid errorLevel, and we are testing that our code protects against it